frontend/src/bundles/common/components/input/input.tsx

```diff
@@ -46,7 +46,7 @@ const Input = <T extends FieldValues>({
     }, []);
 
     const placeholderGenerator = (): string => {
-        return isPassword ? placeholder.repeat(6) : placeholder;
+        return isPassword ? '\u2022'.repeat(6) : placeholder;
```

Comment:
why do we need these changes?

Reply:
Because Dimapopovych said when password type, simply use six bullets as a placeholder without passing it via props.